### ISSUE TITLE
Feature request:  show move details as a table

### ISSUE THREAD

**Issue by raspingbard** [sanderland/katrain#394]
The "Details" tab of the side panel is a bit too wordy for my tastes.  I’d like see the top moves shown as a table, where the move played in the game is highlighted.  The table could also be added as another option next to Details.

If the move played was not included in the top moves (however many are shown), the move would be added as a separate row at the bottom of the table.

<img width="345" alt="Details table" src="https://user-images.githubusercontent.com/28121941/111252214-38f4e080-85d6-11eb-85ff-3e149ad64d5b.png">

Also, the readability of the "Points Loss" tab could be improved.  As it is now, I need to read BOTH the label and the number to understand if the points are gained or lost.  Here’s an example of how it could look using just "–" or "+" signs.

<img width="357" alt="Points - small" src="https://user-images.githubusercontent.com/28121941/111252384-a0129500-85d6-11eb-8b4c-b3f2ee4b79f5.png">

**Comment by raspingbard** [comment]
Here's an example of the table output from zBaduk courtesy of @Marissa.

![Zbaduk](https://user-images.githubusercontent.com/28121941/113517793-68489e80-953f-11eb-807c-44b8b8a42114.png)

**Comment by sanderland** [comment]
I think this is also not anywhere in my priorities.
